app/templates/application.hbs

```diff
@@ -1,16 +1,17 @@
 {{page-title "My | Real Dev Squad"}}
 
+<MobileDialog/>
```

Comment:
Why navbar footer is not rendered in mobile

Reply:
I have it is visible. We need to scroll little bit.
![Screenshot_20231029-141121](https://github.com/Real-Dev-Squad/website-my/assets/56362412/64fd2049-bb16-4872-8cb0-57002ea02bf2)